Makefile

```diff
@@ -36,6 +36,12 @@ distclean:
 	$(RM) -r $(BUILDDIR) $(TARGETS)
 	$(ECHO) " done"
 
+BUSTED_TESTS:=$(wildcard spec/**/*.lua)
```

Comment:
Works for me.
Just try `$(error $(wildcard spec/**/*.lua))` at the top of the Makefile.